spec/features/arclight_spec.rb

```diff
@@ -55,6 +55,8 @@
         expect(page).to have_css('h3 a', text: 'Creator')
         expect(page).to have_css('li .facet-label', text: 'Alpha Omega Alpha', visible: false)
         expect(page).to have_css('h3 a', text: 'Names')
+        expect(page).to have_css('li .facet-label', text: 'Root, William Webster, 1867-1932', visible: false)
```

Comment:
Perhaps we can use the class applied to the panel (which infers the field name) to constrain these tests to ensure that we've met the acceptance criteria.

As the test is written, it would match if the facet value existed in the original facet (which the acceptance criterial explicitly calls out).